app/src/ui/repositories-list/repository-list-item.tsx

```diff
@@ -164,6 +167,10 @@ export class RepositoryListItem extends React.Component<
         action: this.copyToClipboard,
       },
       { type: 'separator' },
+      {
+        label: 'View on GitHub',
+        action: this.viewOnGitHub,
```

Comment:
Need to add a `enabled` property that is only true if this repository can be opened on GitHub. Otherwise, a non-GitHub repo or a local repo will appear like you can click on it, but will not do anything.

The logic we use for toggling the main repository menu's item can be [found here](https://github.com/desktop/desktop/blob/e54cf93459bda29033a72026bab6a48aca823817/app/src/lib/menu-update.ts#L329). A suggestion would be to make a variable like the `missing` one line 158 of this file.